app/routes/achievements/components/Overview.tsx

```diff
@@ -0,0 +1,186 @@
+                }
+                checked={query.min_rarity === '7' && query.max_rarity === 'any'}
+              />
+            </FilterSection>
+            <FilterSection title="Sortering">
+              <RadioButton
+                name="sort"
+                id="sort_none"
+                label="Ingen"
+                onChange={() => setQueryValue('sort')('none')}
+                checked={query.sort === 'none'}
+              />
+              <RadioButton
+                name="sort"
+                id="sort_rarity"
+                label="Sjeldenhet"
+                onChange={() => setQueryValue('sort')('rarity')}
+                checked={query.sort === 'rarity'}
+              />
+              <RadioButton
+                name="sort"
+                id="sort_alphabetical"
+                label="Alfabetisk"
+                onChange={() => setQueryValue('sort')('alphabetical')}
+                checked={query.sort === 'alphabetical'}
+              />
+              <RadioButton
+                name="sort"
+                id="sort_hidden"
+                label="Hemmelig"
+                onChange={() => setQueryValue('sort')('hidden')}
+                checked={query.sort === 'hidden'}
+              />
+            </FilterSection>
+            <FilterSection title="Sorteringsrekkefølge">
+              <RadioButton
+                name="sort_order"
+                id="order_asc"
+                label="Stigende"
+                onChange={() => setQueryValue('sort_order')('asc')}
+                checked={query.sort_order === 'asc'}
+              />
+              <RadioButton
+                name="sort_order"
+                id="order_desc"
+                label="Synkende"
+                onChange={() => setQueryValue('sort_order')('desc')}
+                checked={query.sort_order === 'desc'}
+              />
+            </FilterSection>
+          </>
+        ),
+      })}
+    >
+      <Flex className={styles.listWrapper}>
+        {sortedTrophies.map((e) => (
+          <Flex className={styles.listItem} key={e.name + e.rarity}>
+            <Trophy
+              size={40}
+              color={
+                rarityToColorMap[
+                  AchievementsInfo[e.identifier][e.level].rarity
+                ] ?? 'Gold'
+              }
+              style={
+                AchievementsInfo[e.identifier][e.level].rarity >= 4
+                  ? {
+                      filter: `drop-shadow(0px 0px ${
+                        AchievementsInfo[e.identifier][e.level].rarity - 1
+                      }px ${
+                        rarityToColorMap[
+                          AchievementsInfo[e.identifier][e.level].rarity
+                        ]
+                      })`,
+                    }
+                  : {}
+              }
+            />
+            <Flex column gap="var(--spacing-sm)" className="secondaryFontColor">
+              <h2 className={styles.listItemName}>{e.name}</h2>
```

Comment:
```suggestion
              <h3>{e.name}</h3>
```

The classname does not seem to be used - and it shouldn't be an `h2` since it's just an item.